readthedocs/doc_builder/builder.py

```diff
@@ -290,29 +303,28 @@ def pre_build(self):
 
     def build_html(self):
         commands = []  # self.data.config.build.jobs.build.html
-        for command in commands:
-            self.build_environment.run(command)
+        if commands:
+            for command in commands:
+                self.build_environment.run(command)
```

Comment:
I'm guessing we'll eventually want to capture the return code of these commands, but not important yet. That's definitely a question though (What happens if a command returns non-zero?)

Reply:
Yes. These commands will be logged/displayed and validated as we are currently doing with our commands. So, if they return a non-zero value, the build should fail.